codecov.yml

```
@@ -4,4 +4,13 @@ comment:
-  hide_project_coverage: false
\ No newline at end of file
+  hide_project_coverage: false
+
+coverage:
+  status:
+    patch:
+      default:
+        threshold: 0.25%
+    project:
+      default:
+        threshold: 0.25%
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)